db/migrate/20211211172455_add_user_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] As per the [ERD Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), the column name should be `author` instead of `user` in the `posts` table. Please fix it.

```ruby
create_table :posts do |t|
    ...
    t.references :author, foreign_key: { to_table: 'users' }
  end
```